.env.test

```diff
@@ -1,3 +1,4 @@
 SEED_ADMIN_EMAIL=admin@example.com
 SEED_ADMIN_INITIAL_PASSWORD=password
+CODE_COVERAGE_PERCENTAGE=70
```

Comment:
How did we arrive at this number?

Reply:
Would like it to be `100%` but current coverage when merging this would be `73.56%`.
So I took the current coverage and rounded down to the nearest pretty number.
Do you have a different number in mind?

Reply:
73

Reply:
but 73 is an ugly number 🤔

Reply:
100 is the only good number 🤷

Reply:
Could we separate the two? This PR for moving the existing code coverage helper to its own file, and an other PR to raise code coverage and set the minimum?

Reply:
sure ^__^

Reply:
Extracted to here: https://github.com/abtion/muffi/pull/54